content/warnings/legacy-factories.md

```diff
@@ -56,4 +56,4 @@ function render(MyComponent) {
 
 ## In Depth {#in-depth}
 
-[Read more about WHY we're making this change.](https://gist.github.com/sebmarkbage/d7bce729f38730399d28)
+[Կարդացեք ավելին թե ինչու ենք մենք կատարել այս փոփոխությունները.](https://gist.github.com/sebmarkbage/d7bce729f38730399d28)
```

Comment:
«we're making» -ը ներկա շարունական ա

...թե ինչու ենք մենք կատարում այս փոփոխությունները